packages/tables/src/styled/StyledRow.ts

```diff
@@ -23,36 +23,79 @@ export interface IStyledRowProps {
   size?: ITableProps['size'];
 }
 
+const baseRowColorStyles = ({ theme, isStriped }: IStyledRowProps & ThemeProps<DefaultTheme>) => {
+  return css`
+    border-bottom: ${theme.borders.sm} ${getColor({ variable: 'border.subtle', theme })};
+    background-color: ${isStriped &&
+    getColor({
+      variable: 'background.emphasis',
+      transparency: theme.opacity[100],
+      light: { offset: -300 },
+      theme
+    })};
+  `;
+};
+
 export const StyledBaseRow = styled.tr<IStyledRowProps>`
```

Comment:
Fwiw, this combined `Styled...` components in one file is a bad code smell. Might be good to clean it up here.